MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -2311,6 +2207,7 @@ void serialize(Archive & archive, CompilerMSL::Options& opt) {
 				opt.view_mask_buffer_index,
 				opt.dynamic_offsets_buffer_index,
 				opt.shader_input_buffer_index,
+				opt.shader_patch_input_buffer_index,
 				opt.shader_index_buffer_index,
 				opt.shader_input_wg_index,
```

Comment:
Not trying to be too pedantic here, but can you move this down one line to just before `opt.shader_input_wg_index`, so that it's location aligns with its location in the definition of the struct in `spirv_msl.hpp`, please? It makes it much easier to visually check for required struct members in the Archive definition if they appear in the same order as the struct member definition, wherever possible.

```suggestion
				opt.shader_index_buffer_index,
				opt.shader_patch_input_buffer_index,
				opt.shader_input_wg_index,
```